library/swift/src/GRPCRequestBuilder.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+
+/// Builder used for creating new gRPC `Request` instances.
+@objcMembers
+public final class GRPCRequestBuilder: NSObject {
+  private let underlyingBuilder: RequestBuilder
+
+  /// Initialize a new builder.
+  ///
+  /// - parameter path:      Path of the RPC (i.e., `/pb.api.v1.Foo/GetBar`).
```

Comment:
Should we rename this to rpcPath?

Reply:
I think it's implied by `GRPC`, what do you think?

Reply:
Probably fine